src/main/java/com/problem1/myproject/domain/User.java

```diff
@@ -1,5 +1,38 @@
 package com.problem1.myproject.domain;
 
- 
+import lombok.Data;
+
+import javax.persistence.*;
+import java.util.Date;
+import java.util.List;
+
+//@Entity
+@Data
+@Entity
+@Table(name = "user")
 public class User {
+    enum Roles {
+        ADMIN,
+        USER,
+        GUEST
+    }
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
```

Comment:
@RobertRodeapps do not use this annotation if the java name is identical to db name